diff --git a/data/com.jeffser.Alpaca.metainfo.xml.in b/data/com.jeffser.Alpaca.metainfo.xml.in
index 5d9102d..1a28b69 100644
--- a/data/com.jeffser.Alpaca.metainfo.xml.in
+++ b/data/com.jeffser.Alpaca.metainfo.xml.in
@@ -10,7 +10,6 @@
     <p>A private AI client</p>
     <p>Features</p>
 	  <ul>
-	    <li>Built in Ollama instance</li>
 	    <li>Talk to multiple models in the same conversation</li>
 	    <li>Pull and delete models from the app</li>
 	    <li>Have multiple conversations</li>
