commit 70d6111acc72c81b7473354f4af459e57fdfa098
Author: Jeffser <jeffrysamuer@gmail.com>
Date:   Sat May 24 17:31:07 2025 -0600

    Added env variable for Ollama only mode

diff --git a/src/widgets/instance_manager.py b/src/widgets/instance_manager.py
index ab314b0..de7040f 100644
--- a/src/widgets/instance_manager.py
+++ b/src/widgets/instance_manager.py
@@ -1013,6 +1013,9 @@ def update_instance_list():
         window.instance_listbox.set_selection_mode(1)
         window.instance_listbox.select_row(row)
 
-ready_instances = [OllamaManaged, Ollama, ChatGPT, Gemini, Together, Venice, Deepseek, OpenRouter, Anthropic, Groq, Fireworks, LambdaLabs, Cerebras, Klusterai, GenericOpenAI, LlamaAPI]
+if os.getenv('ALPACA_OLLAMA_ONLY', '0') == '1':
+    ready_instances = [OllamaManaged, Ollama]
+else:
+    ready_instances = [OllamaManaged, Ollama, ChatGPT, Gemini, Together, Venice, Deepseek, OpenRouter, Anthropic, Groq, Fireworks, LambdaLabs, Cerebras, Klusterai, GenericOpenAI, LlamaAPI]
 
 
