diff --git a/src/window.py b/src/window.py
index cd92f49..17366c3 100644
--- a/src/window.py
+++ b/src/window.py
@@ -927,8 +927,8 @@ class AlpacaWindow(Adw.ApplicationWindow):
     def prepare_alpaca(self):
         self.main_navigation_view.replace_with_tags(['chat'])
         # Notice
-        if not SQL.get_preference('last_notice_seen') == self.notice_dialog.get_name():
-            self.notice_dialog.present(self)
+        #if not SQL.get_preference('last_notice_seen') == self.notice_dialog.get_name():
+        #    self.notice_dialog.present(self)
 
         #Chat History
         self.load_history()
